public/app/features/api-keys/ApiKeysPage.tsx

```diff
@@ -221,6 +287,7 @@ export class ApiKeysPage extends PureComponent<Props, any> {
                   <tr key={key.id}>
                     <td>{key.name}</td>
                     <td>{key.role}</td>
+                    <td>{key.expiration}</td>
```

Comment:
Hmm for some reason this comment was not saved last time. But here is the place I think we should add the formatting to current browser local time I think instead of having the formatting server side.

@dprokop is it ok to show times in local time or do we have some other way to deal with time formatting?